core/common/src/main/java/alluxio/security/login/AlluxioLoginModule.java

```diff
@@ -86,11 +66,12 @@ public boolean abort() throws LoginException {
 
   /**
    * Commits the authentication (second phase).
-   *
+   * <p>
    * This method is called if the LoginContext's overall authentication succeeded. (login
    * succeeded)
```

Comment:
The final point must be after the (), what you think?

Reply:
> The final point must be after the (), what you think?

Fixed.